adobe/parliament-transformer-navigation
# Remove vscode settings from project

## Description

Remove and ignore the `.vscode` settings from the project.

## Related Issue

None

## Motivation and Context

These settings are unrelated to how the project is developed or tested.
They only affects the UI, which can conflict with the editor theme a developer chooses to work with.

If a developer wants to use the settings removed from this PR, they may recreate the `.vscode` folder since it is now in the `.gitignore` file.

## How Has This Been Tested?

1. Open project in Visual Studio Code
1. Verify no change in editor theme colors

## Screenshots (if appropriate):

## Types of changes




## Checklist:
